src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,6 +215,34 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
             tableBodyHtml.append(tableRow + Const.EOL);
         }
         
+        StringBuilder mobileHtml = new StringBuilder();
+
+        String mobilePanelTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL;
+        String mobilePanelFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL_FRAGMENT;
+
+        for(int j = 0 ; j < numOfRubricSubQuestions ; j++) {
```

Comment:
Sure, will fix this.
However, I should point out that this different style is being followed at quite a few other places in that file. Examples, line 287 and line 535 for the loop variables and line 287 and line 271 for the space after `for`. We should probably clean these up as a separate issue as it is very inconsistent right now.